py_mmd_tools/mmd_elements.yaml

```
@@ -324,14 +442,44 @@ data_center:
         separator: ','
         comment: Required
         description: The name of the institution principally responsible for originating this data in the format <long institution name> (<short institution name>). This attribute is recommended by the CF convention.
+        format: "text"
+        requirement_level: "Optional"
     long_name:
     short_name:
   data_center_url:
     acdd:
       publisher_url:
         separator: ','
-        comment: Comma separated list.
+        maxOccurs: '1'
+        minOccurs: '1'
+        comment: Required if not hosted by MET. If multiple are listed, use comma separated list and consistency across fields.
         description: The URL of the person (or other entity specified by the publisher_type attribute) responsible for publishing the data file or product to users.
+        format: "text"
+        requirement_level: "Required if not hosted by MET"
+
+# publisher_type is not in MMD but suggested for the netcdf file - is this ACDD?
+not_mmd.publisher_type:
+  maxOccurs: 'unbounded'
+  minOccurs: '0'
+  acdd:
+    publisher_type:
+      maxOccurs: 'unbounded'
+      minOccurs: '0'
+      comment: Suggested
+      description: Specifies type of publisher as one of 'person', 'group', 'institution', or 'position'. If this attribute is not specified, the publisher is assumed to be a person.
+
+# publisher_email is not in MMD but recommended for the netcdf file
+not_mmd.publisher_email:
+  maxOccurs: 'unbounded'
+  minOccurs: '0'
+  acdd:
+    publisher_email:
+      maxOccurs: 'unbounded'
+      minOccurs: '1'
+      comment: Required if not hosted by MET. If multiple are listed, use comma separated list and consistency across fields.
```

```suggestion
      comment: Required. If multiple are listed, use comma separated list and consistency across fields.
```
As above. See, e.g., https://s-enda.pages.met.no/data-management-handbook-met/#default-global-attrs